src/cluster.ts

```diff
@@ -201,6 +239,91 @@ Please use the format 'my-cluster' or '${instance.name}/clusters/my-cluster'.`);
     this.instance.createCluster(this.id, options, callback);
   }
 
+  createBackup(
+    id: string,
+    config: CreateBackupConfig
+  ): Promise<CreateBackupResponse>;
+  createBackup(
+    id: string,
+    config: CreateBackupConfig,
+    callback: CreateBackupCallback
+  ): void;
+  /**
+   * Backup a table from this cluster.
+   *
+   * @param {string} id A unique ID for the backup.
+   * @param {object} config Configuration object.
+   * @param {BackupTimestamp} config.expireTime When the backup will be
+   *     automatically deleted.
+   * @param {string|Table} config.table Table to create the backup from.
+   * @param {CallOptions} [config.gaxOptions] Request configuration options,
+   *     outlined here:
+   *     https://googleapis.github.io/gax-nodejs/CallSettings.html.
+   * @param {CreateBackupCallback} [callback] The callback function.
+   * @param {?error} callback.err An error returned while making this request.
+   * @param {Backup} callback.backup The newly created Backup.
+   * @param {Operation} callback.operation An operation object that can be used
+   *     to check the status of the request.
+   * @param {object} callback.apiResponse The full API response.
+   * @return {void | Promise<CreateBackupResponse>}
+   */
+  createBackup(
+    id: string,
+    config: CreateBackupConfig,
+    callback?: CreateBackupCallback
+  ): void | Promise<CreateBackupResponse> {
+    if (!id) {
+      throw new TypeError('An id is required to create a backup.');
+    }
+
+    if (typeof config !== 'object') {
+      throw new Error('A configuration object is required.');
+    }
+
+    const table = config.table;
+
+    if (!table) {
+      throw new Error('A source table is required to backup.');
+    }
+
+    const reqOpts = {
+      parent: this.name,
+      backupId: id,
+      backup: {
+        sourceTable: typeof table === 'string' ? table : table.name,
+        ...config,
+      },
+    };
+
+    if (reqOpts.backup.expireTime instanceof Date) {
```

Comment:
There used to be all kinds of nuance on when it was ok to use `instanceof Date`, and you had to do weird tricks to detect date objects.  It looks like `is` now does exactly what you're doing here though:
https://github.com/sindresorhus/is/blob/master/source/index.ts

So I'm good with it :)